### ISSUE TITLE
Feature request: exclude rules

### ISSUE THREAD

**Issue by kerim** [DimitryDushkin/logseq-plugin-task-check-date#1]
Thanks for working on this plugin, it is something I've wanted for a long time. But I don't always want it to mark tasks as completed. It would be nice to allow some exclusions, for instance if the block has certain tags present, or even certain kinds of tasks (TODO but not NOW, etc.) One thing I noticed with the version I installed (which isn't the latest one), is that I can't even delete the property unless I disable the plugin.

**Comment by DimitryDushkin** [comment]
> Thanks for working on this plugin, it is something I've wanted for a long time. But I don't always want it to mark tasks as completed. It would be nice to allow some exclusions, for instance if the block has certain tags present, or even certain kinds of tasks (TODO but not NOW, etc.) One thing I noticed with the version I installed (which isn't the latest one), is that I can't even delete the property unless I disable the plugin.

Hey! Glad it’s useful for you!
Please update plugin to latest version. The bug you mentioned is fixed. 

I’ll think about customisation you suggested. I guess it should have a settings page then. Could you please share the workflow where this customisation might be useful?

**Comment by kerim** [comment]
I use all the various task markers: TODO, NOW, DOING, LATER - but I use TODO differently than the others. For me this is for things like shopping lists, daily habits, quick checklists, etc. I don't want any of these marked with the date when I check them as done - only the others. Does that make sense?

**Comment by DimitryDushkin** [comment]
Yeah, thanks for sharing! I'm not planning extending it right now. TBH I'll probably switch back to Obsidian since while I was developing this plugin I got an image that Logseq is not production ready. 
But I'll keep in mind your request.

**Comment by kerim** [comment]
Ah, too bad. I wonder if you could hand over development to someone else? Perhaps @sawhney17 would be willing to take it up? (He was planning on working on a similar feature.)

**Comment by DimitryDushkin** [comment]
@kerim I'll be happy to accept a PR and make a new release :)
